sklearn/preprocessing/data.py

```diff
@@ -1917,3 +1920,38 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(X):
```

Comment:
I waI was hoping to use the Parallel helper here. And couldn't figure how to take only a part of the input while using it. If that is an overkill, will make it a vectorise loop like everywhere else.